### ISSUE TITLE
use tensorboard visualizations

### ISSUE THREAD

**Issue by Tilana** [Tilana/Classification#68]
_From @Tilana on August 24, 2017 17:57_

visualize learning rate, accuracy, and tensor graph (code structure) with tensorboard

_Copied from original issue: Tilana/cnn#3_

**Comment by Tilana** [comment]
Tensorflow Guide: https://www.tensorflow.org/versions/r0.12/how_tos/graph_viz/

**Comment by Tilana** [comment]
tensorboard --logdir=path/to/logfiles
